recipes/osgearth/all/conanfile.py

```diff
@@ -101,10 +105,13 @@ def config_options(self):
     def requirements(self):
 
         self.requires("opengl/system")
-        self.requires("gdal/3.3.1")
+        self.requires("gdal/3.4.3")
         self.requires("openscenegraph/3.6.5")
-        self.requires("libcurl/7.79.1")
+        self.requires("libcurl/7.83.1")
+        self.requires("lerc/2.2")
+        self.requires("rapidjson/1.1.0", private=True)
```

Comment:
```suggestion
        self.requires("rapidjson/1.1.0")
```
Do not use private in Conan Center. Also, is it required for 3.3 only?

Reply:
rapidjson used to be in included the source archive in 3.2, but then became a git submodule in 3.3. So I took the opportunity to use the conan package for both 3.2 and 3.3, same for lerc.
rapidjson is only needed at compile time. So *build_requirements* would be the way to go right?